crates/python/qcs_sdk/qpu/client/__init__.pyi

```diff
@@ -0,0 +1,9 @@
+class QcsClient:
+    """
+    Here we are, deep in the belly of the module
```

Comment:
Haha, I do see your PR description, but it looks like this is still getting worked out.